### ISSUE TITLE
Could not find compiled pymongoarrow.lib extension, please install

### ISSUE THREAD

**Issue by Alan-MTuring** [mongodb-labs/mongo-arrow#96]
I ran into this problem: Could not find compiled pymongoarrow.lib extension, please install.  But I downloaded pymongoarrow.Python version 3.79.

**Comment by blink1073** [comment]
Hi @Alan-MTuring, which operating system are you using?  Can you also give the full traceback?

**Comment by SebastianSchuhbauer** [comment]
Hey,
I have the same problem on Windows 10. Here is the full traceback I get:


C:\Users\Sebi\anaconda3\lib\site-packages\pymongoarrow\__init__.py:34: UserWarning: Could not find compiled pymongoarrow.lib extension, please install from source or report the following traceback on the issue tracker:
Traceback (most recent call last):
  File "C:\Users\Sebi\anaconda3\lib\site-packages\pymongoarrow\__init__.py", line 32, in <module>
    from pymongoarrow.lib import libbson_version
ImportError: DLL load failed while importing lib: Das angegebene Modul wurde nicht gefunden.
  warnings.warn(

Seems like the init function cant find the pymongoarrow.lib file but it is in the pymongoarrow package folder.
Thanks for the help :)

**Comment by Alan-MTuring** [comment]
System: win11, amd; python version: 3.79. In addition, I use another computer: Windows 11,intel; python version: 3.10. It works perfectly. I recommend writing more demo files to make it easier for beginners

**Comment by blink1073** [comment]
I tried the following on win 10 intel 64bit with both Python 3.7 and Python 3.10 downloaded from python.org.  In both cases it worked fine.

```shell
python -m pip install pymongoarrow
python -c "import pymongoarrow.lib"
```

@SebastianSchuhbauer are you on an amd or intel machine?  You can check with `wmic cpu get name`.

Also, it would be helpful to know how you both installed Python in the case where it is not working.

**Comment by SebastianSchuhbauer** [comment]
Hey blink,

I am working on an intel machine and my python 3.9.13 got installed together with Anaconda3 2022.10. When using the code you posted I get for:

`python -m pip install pymongoarrow`

the error mentioned above and if I want to import the lib manually with:

`python -c "import pymongoarrow.lib"`

I get a similar error, the module cannot be found.

**Comment by blink1073** [comment]
I found the underlying problem, and opened https://jira.mongodb.org/browse/ARROW-121 to track the fix.

**Comment by blink1073** [comment]
Closing as fixed by #97.  Please re-open an issue if the error persists after the 0.6.0 release (early next week).
